tests/ZendTest/Mvc/Router/PriorityListTest.php

```diff
@@ -11,6 +11,7 @@
 
 use Zend\Mvc\Router\PriorityList;
 use PHPUnit_Framework_TestCase as TestCase;
+use Zend\Stdlib\ArrayUtils;
```

Comment:
This test should be on [`ChainTest`](https://github.com/zendframework/zf2/blob/1690c8824d0966e564a204bfea1a17d95de6d6d9/tests/ZendTest/Mvc/Router/Http/ChainTest.php), as that's where the code changes are.